client/src/components/ProposalsPage.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { colors } from '../utils';
+import Page from "./Page";
+import {Col, Container, Row} from "reactstrap";
+import {getSessionTypeStr} from "../utils";
+import Tag from './Tag';
+import Speaker from "./Speaker";
+import SpeakerShort from "./SpeakerShort";
+import heroImg from '../images/my-proposals.jpg';
+import TagsList from './TagsList';
+
+const ProposalItem = (proposal, i) => {
+    const {title, type, speaker_ids, tags, abstract} = proposal;
+
+    return <Row key={i}>
+        <Col xs="10" sm={{ size: 7, offset: 1}} className="mb-5">
+            <h4>{title}</h4>
+            <p>{getSessionTypeStr(type)}</p>
+            <div className="d-flex text-muted mb-3">{tags.map(Tag)}</div>
+            <p>{abstract}</p>
+        </Col>
+        <Col xs="2" sm="3" className="mb-5 ml-4">
+            {speaker_ids.map(SpeakerShort)}
```

Comment:
well done on using `SpeakerShort`.
But i think it's not behaving perfectly in vertical scenario. Might need to use another component, or pass some prop like `vertical`